src/web/app/components/sessions-table/sessions-table.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="row" *ngIf="sessionsTableRowModels.length > 0; else noSessionMessage">
-  <div class="col-12">
+<div *ngIf="sessionsTableRowModels.length > 0; else noSessionMessage">
+  <div>
```

Comment:
This `<div>` can likely be removed entirely now that there are no styles attached to it. Moving forward, feel free to remove any unnecessary `<div>`s as we go along.